test/fixtures/webpack-test-server.ts

```diff
@@ -5,15 +5,13 @@ import * as serve from 'webpack-serve';
 
 // tslint:disable:no-console
 
-// TODO: It should be nice to switch here to webpack-dev-server, as for manual running, but that
-// one does not work properly out of the box (in this context), whereas webpack-serve does.
 export class WebpackServer implements IMochaServer {
   // The result of webpack-serve call. See https://github.com/webpack-contrib/webpack-serve#serveargv-options
   private _server: any;
 
   public async start() {
     const config = require(path.resolve(__dirname, 'webpack.config.js'));
-    console.log("Starting webpack-serve");
+    console.error("Starting webpack-serve");
```

Comment:
You want these logged as errors?

Reply:
Forgot to respond -- I copied this from Grist's set up. Console.errror outputs to stderr, while console.log goes to stdout; stderr is better because it doesn't interfere without test output. It really only matters for xml output (e.g for Jenkins expects). In this repo it doesn't matter, but I kept it in case it matters in the future.